.changeset/many-tools-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: custom message formatting to `custom-media-pattern`, `custom-property-pattern`, `keyframes-name-pattern`, `selector-class-pattern`, and `selector-id-pattern`
```

Comment:
[ask] Too long?

Reply:
```suggestion
Added: `*-pattern` custom message formatting
```
To be consistent with the likes of "Added: `font-weight-notation` autofix'".

Reply:
Nice! Thanks.